client/src/commands.ts

```diff
@@ -8,13 +8,14 @@
 
 import * as vscode from 'vscode';
 import * as lsp from 'vscode-languageclient';
+import {ServerOptions} from '../../common/out/initialize';
 
 /**
  * Represent a vscode command with an ID and an impl function `execute`.
  */
 interface Command {
   id: string;
-  execute(): Promise<vscode.Disposable>;
+  execute(): Promise<void|vscode.Disposable>;
```

Comment:
```suggestion
  execute(): unknown;
```

Since we never actually use this return type